examples/use-cases/Kernel2Memif/README.md

```diff
@@ -32,34 +32,37 @@ kind: Kustomization
 
 namespace: ${NAMESPACE}
 
+resources: 
+- client.yaml
 bases:
-- https://github.com/networkservicemesh/deployments-k8s/apps/nsc-kernel?ref=5012a8aafd293534e2a9d98903f6d339ef44ceab
 - https://github.com/networkservicemesh/deployments-k8s/apps/nse-memif?ref=5012a8aafd293534e2a9d98903f6d339ef44ceab
 
 patchesStrategicMerge:
-- patch-nsc.yaml
 - patch-nse.yaml
 EOF
 ```
 
-Create NSC patch:
+Create Client:
 ```bash
-cat > patch-nsc.yaml <<EOF
+cat > client.yaml <<EOF
 ---
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: v1
+kind: Pod
 metadata:
-  name: nsc-kernel
+  name: alpine
+  labels:
+    app: alpine    
+  annotations:
+    networkservicemesh.io: kernel://icmp-responder/nsm-1
 spec:
-  template:
-    spec:
-      containers:
-        - name: nsc
-          env:
-            - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder/nsm-1
-      nodeSelector:
-        kubernetes.io/hostname: ${NODE}
+  containers:
+  - name: alpine
+    image: alpine
```

Comment:
Please use a concrete alpine image.
Also, add the alpine image into [external-images.yaml](https://github.com/networkservicemesh/deployments-k8s/blob/main/external-images.yaml).